parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -58,6 +89,11 @@ installDeps() {
             exit $ERR_APT_INSTALL_TIMEOUT
         fi
     done
+
+    if [ "${UBUNTU_RELEASE}" == "18.04" ]; then
+        # curl version on 18.04 is buggy when targetting http/2 TLS endpoint, manually installing a newer version
+        installLatestCurlManuallyIfNotPresent
```

Comment:
maybe we should wrap this in an event so we can have direct reporting on the latency this step incurs during provisioning? would most likely be helpful in cases where we unexpectedly see this step trying to install a new version of curl at runtime

I see we already wrap installDeps in an event as called from cse_main.sh, though having more granular telemetry here would be helpful I think